yt/yt/server/queue_agent/queue_static_table_exporter.cpp

```diff
@@ -330,7 +339,7 @@ class TQueueExportTask
         if (QueueObject_.Type != EObjectType::Table) {
             THROW_ERROR_EXCEPTION(
                 "Invalid type of %v: expected %Qlv, %Qlv found",
-                QueueObject_.GetPath(),
+                Queue_,
```

Comment:
(idk how to add comments to unchanged lines)
Maybe `YT_VERIFY` that object id returned from attributes is equal to our saved object id in path?
Probably easier the other way around: `FromObjectId(objId) = GetPath()`.